layers/+lang/c-c++/packages.el

```diff
@@ -41,23 +41,27 @@
     srefactor
     xcscope
     ;; lsp
-    (ccls :requires lsp-mode)
+    (ccls :requires lsp-mode :toggle (eq c-c++-backend 'lsp-ccls))
     dap-mode
     ;; rtags
-    (company-rtags :requires (company rtags))
-    (flycheck-rtags :requires (flycheck rtags))
+    (company-rtags :requires (company rtags) :toggle (eq c-c++-backend 'rtags))
+    (flycheck-rtags :requires (flycheck rtags) :toggle (eq c-c++-backend 'rtags))
```

Comment:
These packages already depend on rtags which is owned by this layer, so I think these toggles are probably redundant.